src/pages/Nav.js

```diff
@@ -1,18 +1,19 @@
 import React from "react";
-import "../css/nav.css"
+import "../css/nav.css";
 
-import { Route,Routes,Link } from "react-router-dom";
+import { Route, Routes, Link } from "react-router-dom";
 import Home from "./Home";
 import Docs from "./Docs";
 import Label from "./Label";
 import Login from "./Login";
 import Notes from "./Notes";
-import Remainder from "./Remainder";
-import Trash from "./Trash"
+import Archive from "./Archive";
+import Trash from "./Trash";
```

Comment:
You can import all these in one file , export from that single file and then import it here.